tests/suites/test_suite_x509write.data

```diff
@@ -59,19 +59,22 @@ depends_on:MBEDTLS_SHA1_C:MBEDTLS_RSA_C:MBEDTLS_PKCS1_V15:MBEDTLS_DES_C:MBEDTLS_
 x509_crt_check:"data_files/server1.key":"":"C=NL,O=PolarSSL,CN=PolarSSL Server 1":"data_files/test-ca.key":"PolarSSLTest":"C=NL,O=PolarSSL,CN=PolarSSL Test CA":"1":"20110212144406":"20210212144406":MBEDTLS_MD_SHA1:0:0:MBEDTLS_X509_CRT_VERSION_1:"data_files/server1.v1.crt"
 
 X509 String to Names #1
-mbedtls_x509_string_to_names:"C=NL,O=Offspark\, Inc., OU=PolarSSL":"C=NL, O=Offspark, Inc., OU=PolarSSL":0
+mbedtls_x509_string_to_names:"C=NL,O=Offspark\, Inc., OU=PolarSSL":"C=NL, O=Offspark\, Inc., OU=PolarSSL":0
 
 X509 String to Names #2
+mbedtls_x509_string_to_names:"C=NL,O="Offspark, Inc.", OU=PolarSSL":"C=NL, O="Offspark, Inc.", OU=PolarSSL":0
+
+X509 String to Names #3
```

Comment:
Perhaps instead of changing all the test numbers you could simply add new tests with higher numbers that currently do not exist? e.g. 7, 8, 9, etc. Reducing the number of changes makes the important code changes easier to see.